### DISCUSSION TITLE
Is it possible to query and set the time on the controller?

### DISCUSSION THREAD

**Discussion by gyrex** [strandborg/AlsavoCtrl#21] [Q&A]
Hi,

I was wondering if it's possible to query the time on the controller and if it's also possible to set the time of the controller? I can't see any way to do it via Alsavo Pro. The only way I can see to do this is via the actual LCD panel which seems crazy to me.

**Comment by strandborg** [comment]
I think the config param #32 can be used to set the current time, with a 16-bit integer where the highest 8 bits = hours and lowest 8 bits = minutes; however, experiment on your own risk on this one, as even the Alsavo app cannot be used to set the clock.

**Reply by gyrex** [reply]
Thanks mate. It's pretty strange that they haven't allowed the clock to be set via the app or parsing a value...
